kotlin/workflow-core/src/main/java/com/squareup/workflow/Worker.kt

```diff
@@ -184,6 +187,11 @@ interface Worker<out T> {
       block: suspend () -> Unit
     ): Worker<Nothing> = TypedWorker(Nothing::class, key, flow { block() })
 
+    /**
+     * Returns a [Worker] that finishes immediately without emitting anything.
+     */
+    fun finished(): Worker<Nothing> = FinishedWorker
```

Comment:
Unused? Should at least be covered by a smoke test.

Reply:
Everything needs unit tests here.

Reply:
It's not unused, see GameWorkflow above, line 105.

I'm wondering if this should be a function though, or we should just expose it as a val. No need to do any casting here so no requirement to have it as a function.

Reply:
+1 to making it a val

Reply:
I'm going to keep it a function to be consistent with what Kotlin does: `emptyFlow()` is a function. I'm also going to give it a generic, so you can say `Worker.finished<String>()`.

https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/empty-flow.html